packages/util/src/environment.ts

```diff
@@ -173,6 +173,12 @@ export function isSafari(): boolean {
   );
 }
 
+export function isChrome(): boolean {
+  return (
+    !isNode() && !!navigator.userAgent && navigator.userAgent.includes('Chrome')
+  );
+}
+
```

Comment:
This will return true for all Chromium-based browsers, which include `Chrome` in their UA string (Edge, Brave, Opera, etc). For example, Brave's UA string is something like `'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36'`, so this function would incorrectly return `true` in Brave.

Firebase Auth implemented an accurate [`_getBrowserName`](https://github.com/firebase/firebase-js-sdk/blob/main/packages/auth/src/core/util/browser.ts#L49) that we could use instead if it were moved to `firebase/util`.

Reply:
Thank you for reviewing!

Reply:
Copied this feedback into our burndown.